domain/src/main/java/com/safetyheads/akademiaandroida/domain/usecases/UpdateProfileFCMUseCase.kt

```diff
@@ -0,0 +1,14 @@
+package com.safetyheads.akademiaandroida.domain.usecases
+
+
+import com.safetyheads.akademiaandroida.domain.repositories.UserRepository
+import kotlinx.coroutines.flow.Flow
+
+class UpdateProfileFCMUseCase(private val repository: UserRepository) :
+    BaseUseCase<UpdateProfileFCMUseCase.UpdateProfileParam, Unit> {
```

Comment:
You can use here `ParameterlessBaseUseCase`

Reply:
This usecase requires UserUUID and FcmToken which are passed via UpdateProfileParam. If i use ParameterlessBaseUseCase i cannot pass any data